### PR TITLE
Jl support

### PR DESCRIPTION
**Pull request by AShedko**
Macro support and other function definition syntax.

### CODE REVIEW

**Comment by AShedko** [comment]
Fixed conflicts.

File: dumb-jump.el [hunk 1]
```diff
@@ -883,6 +891,7 @@ or most optimal searcher."
     (:language "javascript" :ext "vue" :agtype "js" :rgtype "js")
     (:language "javascript" :ext "html" :agtype "html" :rgtype "html")
     (:language "javascript" :ext "css" :agtype "css" :rgtype "css")
+    (:language "julia" :ext "jl" :agtype "jl" :rgtype "jl")
```
Comment (jacktasia) [inline dumb-jump.el]: We don't need this.

**Comment by jacktasia** [comment]
Thanks for the PR! This looks good. I'll merge it upon removing the extra julia extension def (see comment).

**Comment by jacktasia** [comment]
Thanks!

**Pull request merged**